usaspending_api/api_contracts/contracts/v2/autocomplete/awarding_agency_office.md

```diff
@@ -0,0 +1,72 @@
+FORMAT: 1A
+HOST: https://api.usaspending.gov
+
+# Awarding Agency and Office Autocomplete [/api/v2/autocomplete/awarding_agency_office/]
+
+This endpoint can be used to autocomplete Awarding Agency and Office searches on the Advanced Search page. It will return
+any agencies, sub-agencies, and offices that match the search text. Additionally, any matching agencies will include associated
+offices and sub-agencies, any matching sub-agencies will include associated agencies and offices, and any matching offices will include
+associated agencies and sub-agencies.
+
+## POST
+
+This route sends a request to the backend to retrieve awarding agencies and offices matching the specified search text.
+
++ Request (application/json)
+    + Schema
+
+            {
+                "$schema": "http://json-schema.org/draft-04/schema#",
+                "type": "object"
+            }
+
+    + Attributes (object)
+        + `limit` (optional, number)
+        + `search_text` (required, string)
+    + Body
+
+            {
+                "search_text": "Defense"
```

Comment:
Will the endpoint return results if the agency abbreviations is searched?  Or is it only based text in the agency's full name?

Reply:
Yes, I’m asking if I were to send a request where the “search_text” is an agency abbreviation such as DOT would it return results for the department of treasury? I’m asking because abbreviation is an attribute of toptier_agency per the contract.

Reply:
I can make that happen. Is that what you want?

Reply:
Right now it's only going by agency name

Reply:
Ok I was just checking.  I can check with the designers about that